.github/workflows/python_bindings.yml

```diff
@@ -0,0 +1,263 @@
+name: Python Bindings
+
+on:
+  push:
+    branches: [ master, develop ]
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+  pull_request:
+    branches: [ master, develop ]
```

Comment:
This is going to need tweaking. I'm still not sure when we should build the python bindings or not.

Reply:
master and develop should be good for now.  If a user needs a specific branch built, something like this  can add this to that action .yaml file. 

on:
  push:
    branches:
      - test_branch